api/specs/common/schemas/project-v0.0.1-converted.yaml

```diff
@@ -407,7 +407,9 @@ properties:
                   - 0
                   - 2
               instructions:
-                type: string
+                type:
+                  - string
```

Comment:
these changes are in the other PR, right?

Reply:
Yes https://github.com/ITISFoundation/osparc-simcore/pull/3593